### PR TITLE
List MonetaryUnit (MUE)

### PR DESCRIPTION
**Pull request by sotblad**
- Official project URL: https://monetaryunit.org
- Official block explorer URL: https://chainz.cryptoid.info/mue

### CODE REVIEW

File: src/test/java/bisq/asset/coins/MonetaryUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+ package bisq.asset.coins;
+ import bisq.asset.AbstractAssetTest;
+ import org.junit.Test;
+ public class MonetaryUnitTest extends AbstractAssetTest {
```
Comment (blabno) [inline src/test/java/bisq/asset/coins/MonetaryUnitTest.java:20]: Does not conform to Bisq codestyle.

File: src/main/java/bisq/asset/MonetaryUnit.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+ package bisq.asset.coins;
```
Comment (blabno) [inline src/main/java/bisq/asset/MonetaryUnit.java:17]: Wrong directory!
Does not conform to Bisq codestyle.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

File: src/main/java/bisq/asset/MonetaryUnit.java (hunk 2 above)
Comment (sotblad) [reply src/main/java/bisq/asset/MonetaryUnit.java:17]: Jesus Christ, sorry. #112 should be fine.

**Pull request closed without merging**